### ISSUE TITLE
ldflags default includes Commit instead of ShortCommit

### ISSUE THREAD

**Issue by leighmcculloch** [goreleaser/goreleaser#1264]
**Describe the bug**

When building using the default `ldflags` the `commit` variable is set with a long full commit SHA (`Commit`) instead of the truncated short commit sha. The docs say that the default ldflags sets `main.commit` to the `ShortCommit`.

The docs at https://goreleaser.com/build/ state that this is the default ldflags:
```yaml
    # Custom ldflags templates.
    # Default is `-s -w -X main.version={{.Version}} -X main.commit={{.ShortCommit}} -X main.date={{.Date}} -X main.builtBy=goreleaser`.
    ldflags:
     - -s -w -X main.build={{.Version}}
     - ./usemsan=-msan
```

The commit that first introduced the default value for the `main.commit` variable is 4af2cb00ea49d9b4544ea42becb741d2cde00d8d from 3 years ago. It was introduced using `Commit` and not `ShortCommit`.

This appears to be either a long-lived bug that has existed since the default was introduced, or incorrect documentation.

**To Reproduce**

You can reproduce this behavior with a simple repository with little code:

```
$ mkdir test
$ cd test
$ git init
$ git remote add origin git@github.com:leighmcculloch/test
$ cat <<-EOF >main.go
package main
import "fmt"
var commit = ""
func main() {
        fmt.Println(commit)
}
EOF
$ cat <<-EOF >.goreleaser.yml
builds:
  - binary: test
    goos:
      - linux
    goarch:
      - amd64
EOF
$ git add . && git commit -m 'Initial commit' && git tag v0.0.1
$ goreleaser --skip-publish
$ ./dist/test_linux_amd64/test 
004db4679806fa69a57bb6a27f693d0c632e5fcd
```
When configuring the ldflags manually with `ShortCommit` it appears to work as intended:
```
$ cat <<-EOF >>.goreleaser.yml
    ldflags:
      -s -w -X main.commit={{.ShortCommit}}
EOF
$ git add . && git commit -m 'Add custom ldflags' && git tag v0.0.2
$ goreleaser --skip-publish
$ ./dist/test_linux_amd64/test 
6b4e614
```

**Expected behavior**

The value set for the `commit` global variable be a short commit sha as implied by the name of the template variable `.ShortCommit`. e.g. `004db46`.

**Environment (please complete the following information):**

- OS: [e.g. mac, linux]
   ```
   Linux
   ```
- OS version
   ```
   Linux devenv-0 4.9.184-linuxkit #1 SMP Tue Jul 2 22:58:16 UTC 2019 x86_64 GNU/Linux`
   ```
- GoReleaser Version
   ```
   version: 0.117.2
   commit: 2c08ab87a13ce4a70cf89a691fbbc46c7da6c2e6
   built at: 2019-09-06T18:55:02Z
   built by: goreleaser
   ```

**Additional context**
I discovered the issue in this CI build where the final output of the version included a long commit instead of a short commit:
```
gas 3.0.2 6e47159666eebcfea2e1226c3efb44993a211305 2019-11-27T20:34:34Z
```
https://github.com/leighmcculloch/gas/commit/6e47159666eebcfea2e1226c3efb44993a211305/checks?check_suite_id=332155089

**Comment by leighmcculloch** [comment]
I've opened a suggested fix to the docs in #1265.

**Comment by caarlos0** [comment]
thanks so much @leighmcculloch 🚀
